recipes/openssh/meta.yaml

```
@@ -0,0 +1,51 @@
+package:
+  name: openssh
+  version: 7.9p1
+
+source:
+  url: http://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-7.9p1.tar.gz
+  sha256: 6b4b3ba2253d84ed3771c8050728d597c91cfce898713beb7b64a305b6f11aad
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - autoconf
+    - pkg-config
+    - {{ compiler('c') }}
+  host:
+    - zlib
+    - libedit
+    - krb5
+    - openssl
+
+test:
+  commands:
```

Maybe add tests which use some of these commands such as:
```suggestion
  commands:
    - ssh-keygen -t rsa -b 4096 -f id_rsa -P do-not-pass-passwords
```